### ISSUE TITLE
ui-file dropzone not stretching to fill column width

### ISSUE THREAD

**Issue by SvetlozarValchev** [adarshpastakia/aurelia-ui-framework#227]
There seems to be a difference between the example provided here: https://aurelia-ui-framework.herokuapp.com/docs/form and the actual implementation

The dropzone doesn't stretch to fill the column width. It seems that getting it out of `<input-wrapper>` (like it is in the example docs) fixes it.

![image](https://user-images.githubusercontent.com/446056/78068177-a3e6e080-73a0-11ea-9d15-dd33dc6091a3.png)

**Comment by SvetlozarValchev** [comment]
Found the issue. The @viewResources is not working for some reason and doesn't get included in ui-file, ui-input, etc. Which means it has no special behaviour and therefore there's no @containerless to remove it, so it breaks the CSS.  

This happens only when included as a module, the reason it works in the Demo Docs is because it's directly loaded into Aurelia.

**Comment by firstfocus** [comment]
@SvetlozarValchev did you find any solution to this? I am having issues with all custom elements with @viewResources when I do a local compile of this library to dist and use npm link.

**Comment by SvetlozarValchev** [comment]
@firstfocus Not really, I don't remember if I either solved it with CSS override or loading it the way it's in the demo (directly loading it in), but I eventually moved to a different UI framework as this one seems abandoned.

**Comment by firstfocus** [comment]
@SvetlozarValchev thanks for the reaction! Which framework did you switch to? I'm struggling finding an alternative as complete as this.

**Comment by SvetlozarValchev** [comment]
@firstfocus Tried Materialize and a few others, but ultimately switched to Vue.js + Vuetify as most UI Frameworks for Aurelia seem to be either abandoned or insufficient for our needs.

**Comment by firstfocus** [comment]
@SvetlozarValchev Looks like we will be switching too. Thanks for the tip!
